### ISSUE TITLE
Support Dart 2.1.0

### ISSUE THREAD

**Issue by sir-boformer** [brianegan/reselect_dart#4]
The latest master version of flutter uses Dart 2.1.0.

reselect_dart itself supports Dart 2.1.0, but memoize does not.

```
The current Dart SDK version is 2.1.0-dev.0.0.flutter-be6309690f.

Because reselect >=0.3.0 depends on memoize >=1.1.0 which requires SDK version >=1.21.0 <2.0.0, reselect >=0.3.0 is forbidden.
So, because timmer depends on reselect ^0.3.0, version solving failed.
pub upgrade failed (1)
```

**Comment by long1eu** [comment]
https://github.com/brianegan/reselect_dart/pull/5

**Comment by brianegan** [comment]
Thanks for the report and the fix, @long1eu! New version that fully supports Dart 2 published as `0.4.0`.
